docs/index.rst

```diff
@@ -0,0 +1,112 @@
+.. Couchbase Python Client Library documentation master file, created by
+   sphinx-quickstart on Thu Apr 14 13:34:44 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+===========================================================
+Welcome to the Couchbase Python Columnar SDK documentation!
+===========================================================
+
+Getting Started with the Python Columnar SDK
+---------------------------------------------
+
+:doc:`using_the_columnar_sdk`
+   Useful information for getting started and using the Python Columnar SDK.
+
+Synchronous API
+---------------
+
+:doc:`couchbase_columnar_api/columnar_core`
+   API reference for Cluster, Database and Scope objects.
+
+:doc:`couchbase_columnar_api/query`
+   API reference for query (SQL++) operations.
+
+Global API
+-----------------
+:doc:`couchbase_columnar_api/credential`
+   API reference for Credential.
+
+:doc:`couchbase_columnar_api/errors`
+   API reference for Errors.
+
+:doc:`couchbase_columnar_api/options`
+   API reference for operation options.
+
+:doc:`couchbase_columnar_api/results`
```

Comment:
This makes `BlockingQueryResult` appear in the Global API, which is synchronous-API specific

Reply:
Updated 👍